packages/db/eslint.config.js

```diff
@@ -1,10 +1,4 @@
-import baseConfig, { restrictEnvAccess } from "@acme/eslint-config/base";
+import baseConfig from "@acme/eslint-config/base";
 
 /** @type {import('typescript-eslint').Config} */
-export default [
-  {
-    ignores: ["dist/**"],
-  },
-  ...baseConfig,
-  ...restrictEnvAccess,
```

Comment:
this was removed by accident?

Reply:
Do you mean `restrictEnvAccess`? If yes then should it be there if we're not using t3-env? If no then the rest is the same as before now. Should I add `restrictEnvAccess` back?

Reply:
i thought we were using t3-env here